code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -5,8 +5,6 @@
 	icon_state = "cleaner"
 	item_state = "cleaner"
 	flags = OPENCONTAINER|NOBLUDGEON
-	//TFF 24/12/19 - Let people print more spray bottles if needed.
-	matter = list("glass" = 300, DEFAULT_WALL_MATERIAL = 300)

```

Comment:
Was this intentional? This will make the spray bottles printable for free.

Reply:
..That was not. Been having some github issues recently and it's starting to be concerning.